.github/workflows/test.yml

```diff
@@ -159,16 +170,6 @@ jobs:
           cat 'cabal.project.freeze' && \
           echo '' || \
           echo 'WARNING: Could not produce the `freeze`.'
-          echo ::set-output name=value::${{ hashFiles('cabal.project.freeze') }}
-
```

Comment:
Can you recap on this detail, by "resolving `${{ hashFiles( ... ) }}` before the step", you likely meant this use to produce output inside the step.

Reply:
https://github.com/haskell/haskell-language-server/pull/2562#issuecomment-1004776728 pretty much added additional context to understand the message.